document-generation/igce-document.ts

```diff
@@ -245,9 +245,19 @@ export async function generateIGCEDocument(
   const surgeCapabilities = summarySheet.getCell("A23");
   surgeCapabilities.value = payload.surgeCapabilities / 100;
 
-  // DITCO Fee
-  const ditcoFee = summarySheet.getCell("C26");
-  ditcoFee.value = { formula: `=K24 *.0225`, date1904: false };
+  // DITCO Fee / Contracting Shop Fee
+  const contractingShopName = summarySheet.getCell("B26");
+  const contractingShopFee = summarySheet.getCell("C26");
+  if (payload.contractingShop.name === "OTHER") {
+    if (payload.contractingShop.fee > 0) {
+      const fee = (payload.contractingShop.fee / 100).toFixed(2); // round to 2 decimal places
+      contractingShopName.value = "Contracting Office Fee";
+      contractingShopFee.value = { formula: `=K24 *${fee}`, date1904: false };
+    }
+  } else if (payload.contractingShop.name === "DITCO") {
+    contractingShopName.value = "DITCO Fee";
+    contractingShopFee.value = { formula: `=K24 *.0225`, date1904: false };
```

Comment:
looks good.

Reply:
Label change as expected as well as fee calculation for non-DITCO contracting shop